mitodl/open-discussions
# new config file format

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2369

#### What's this PR do?
This pr updates the youtube video extract() function to use a more user friendly configuration repo format
#### How should this be manually tested?
Set
```
 OPEN_VIDEO_DATA_BRANCH=format-change-2369
```
in your .env

in the console run

```
from course_catalog.etl import youtube
g = youtube.extract()
next(g)
```
This should step through raw youtube api results for the videos for the channels int he open-video-data-repor.

The code currently throws an error for the master branch because it assumes that there is a `youtube` folder in the `open-video-data` repo.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
